bokeh/models/graphs.py

```diff
@@ -33,22 +31,9 @@ class StaticLayoutProvider(LayoutProvider):
         }
     """)
 
-
-_DEFAULT_NODE_SOURCE = lambda: ColumnDataSource(data=dict(index=[]))
-
-
-_DEFAULT_EDGE_SOURCE = lambda: ColumnDataSource(data=dict(start=[], end=[]))
-
-
-class GraphSource(Model):
-    '''
-
-    '''
-
-    @classmethod
-    def from_networkx(cls, graph, layout_function, **kwargs):
+def from_networkx(graph, layout_function, **kwargs):
```

Comment:
I'd be little concerned about just "networkx" because I could imagine users squashing their networkx package import, but I agree that we can decide before the release.